include/envoy/http/filter.h

```diff
@@ -28,9 +28,13 @@ namespace Http {
 enum class FilterHeadersStatus {
   // Continue filter chain iteration.
   Continue,
-  // Do not iterate to any of the remaining filters in the chain. Returning
+  // Do not iterate for headers to any of the remaining filters in the chain. Returning
```

Comment:
I think I'm arguing semantics so feel free to ignore, but I think "Do not iterate for headers to any of the remaining filters in the chain" sounds like data and trailers will be (but data and trailers are also not iterated to any of the remaining filters).

Maybe just move the clarification to the second paragraph too?
````
Do not iterate to any of the of remaining filters in the chain....

Note that data will still be processed for the current filter unless a local reply is sent. If a local reply is sent, no further iteration for headers as well as data and trailers for the current filter and the filters following....
```

Reply:
Ok, just pushed something... not quite what you suggested but hopefully better :-)

Reply:
Yeah, the statement holds true anyway, and it would be bizarre to come to a different conclusion :)